client/executor/wasmtime/src/instance_wrapper.rs

```diff
@@ -462,3 +420,23 @@ impl InstanceWrapper {
 		}
 	}
 }
+
+impl runtime_blob::InstanceGlobals for InstanceWrapper {
+	type Global = wasmtime::Global;
+
+	fn get_global(&self, export_name: &str) -> Self::Global {
+		self.instance
+			.get_global(export_name)
+			.expect("get_global is guaranteed to be called with an export name of a global; qed")
```

Comment:
How is that guaranteed?

Reply:
This is responsibility of the caller. The contract of `InstanceGlobals::get_global` states:

> The requested export is must exist in the exported list, and it should be a mutable global.